CMakeLists.txt

```diff
@@ -37,9 +37,9 @@ message(STATUS "\n\n-- ====== Finding Dependencies ======")
 
 #--------------------------------------
 # Find gz-common
-gz_find_package(gz-common5
```

Comment:
we are scaling an optional dependency to a mandatory dependency here. I would prefer not to do it in the stable series since it can cause packaging problems in downstream distros or other building problems.

As an alternative, we can leave the current code commented for the main branch and add a private function to `src/InstallationDirectories.cc` by duplicating the `joinPaths` code from common in that file.